src/components/Layout/Layout.module.css

```diff
@@ -5,6 +5,7 @@
   }
   
   .content > :first-child {
+    flex: 1;
```

Comment:
the flex needs to be on `.content` directly to hold the space open on short pages

Reply:
```
.content {
    flex: 1;
}
  
.content > :first-child {
    ...
}
```